Lib/test/test_long.py

```diff
@@ -956,6 +956,16 @@ def test_huge_rshift_of_huge(self, size):
         self.assertEqual(huge >> (sys.maxsize + 1), (1 << 499) + 5)
         self.assertEqual(huge >> (sys.maxsize + 1000), 0)
 
+    @support.cpython_only
+    def test_small_ints_in_huge_calculation(self):
+        a = 2 ** SHIFT
```

Comment:
Does this number fit into a single digit or requires two digits? Maybe add a comment to explain that these numbers use more than 1 digit internally.

Reply:
I switched to `2 ** 100`, which is more intuitive.